plugins/MauticCrmBundle/Integration/ConnectwiseIntegration.php

```diff
@@ -317,13 +317,30 @@ public function appendToForm(&$builder, $data, $formArea)
         if ($formArea == 'integration') {
             if ($this->isAuthorized()) {
                 $builder->add(
-                    'campaign_task',
-                    'integration_campaign_task',
+                    'push_activities',
+                    'yesno_button_group',
                     [
-                        'label'  => false,
-                        'helper' => $this->factory->getHelper('integration'),
-                        'data'   => (isset($data['campaign_task'])) ? $data['campaign_task'] : [],
-                    ]);
+                        'label'      => 'mautic.plugin.config.push.activities',
+                        'label_attr' => ['class' => 'control-label'],
+                        'attr'       => [
+                            'class'    => 'form-control',
+                            'onChange' => 'Mautic.pushConnectwiseActivities(this)',
+                        ],
+                        'required' => false,
+                    ]
+                );
+                $hideClass = (isset($data['push_activities']) && !empty($data['push_activities'])) ? '' : ' hide';
```

Comment:
Instead of using this, use `'data-hide-on' => '{"campaignevent_properties_config_push_activities_0":"checked"}` in the `attr` array for each of the activity specific fields.